scripts/lang/en/places.py

```diff
@@ -0,0 +1,2425 @@
+placetypes = {
```

Comment:
Could you add a docstring as it is done for `langs.py`? And you can directly add the good Wiki page for the linking with your script.